Classes/KIFSystemTestActor.h

```diff
@@ -10,7 +10,11 @@
 #import "KIFTestActor.h"
 #import <UIKit/UIKit.h>
 
+#ifdef __cplusplus
+#define systemTester KIFActorWithClass(KIFSystemTestActor)
+#else
 #define system KIFActorWithClass(KIFSystemTestActor)
```

Comment:
Maybe we should instead do:

```
#define systemTester KIFActorWithClass(KIFSystemTestActor)
#ifndef __cplusplus
// The symbol `system` collides with the cstdlib for compiling C++. Leaving it available for compatibility reasons.
// This will be removed with the next major KIF release, please start using `systemTester` instead.
#define system KIFActorWithClass(KIFSystemTestActor)
#endif
```

That way, we can align around a single definition in the future and switch over all KIF usages from `system` to `systemTester`. Thoughts?